tests/tt_metal/tt_metal/unit_tests_common/gtest_smoke/test_brisc_basic.cpp

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+
+// #include <algorithm>
+// #include <functional>
+// #include <random>
+
+#include "tests/tt_metal/tt_metal/unit_tests_common/common/dprint_fixture.hpp"
+#include "tests/tt_metal/tt_metal/unit_tests_common/common/test_utils.hpp"
+#include "tt_metal/detail/tt_metal.hpp"
+#include "tt_metal/host_api.hpp"
+
+using namespace tt;
+
+namespace gtest_smoke::test_brisc_basic{
+
+bool test_brisc_dprint(DPrintFixture *fixture, tt_metal::Device *device){
```

Comment:
not sure why dprint tests are here?

Reply:
you're right.. the brisc_basic.cpp tests are removed as similar tests are run in device.cpp which is already included in the test suite